src/modeler/AutoRest.Swagger/Validation/Core/JsonValidationLogListener.cs

```diff
@@ -27,7 +27,7 @@ public void Log(LogMessage message)
                     .FirstOrDefault();
                 var pathComponents = AutoRest.Swagger.Model.Utilities.ValidationUtilities.ResourcePathPattern.Match(path ?? "");
                 var pathComponentProviderNamespace = pathComponents.Groups["providerNamespace"];
-                var pathComponentResourceType = pathComponents.Groups["resourceType"];
+                var pathComponentResourceType = pathComponents.Groups["resource"];
```

Comment:
are the tests not enough to verify the code? if not, please add any relevant tests.